### PR TITLE
Basic Resume using html

### PR DESCRIPTION
**Pull request by DeliaMDsouza**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1 +1,109 @@
...
+          <tr>
+            <td>SSC</td>
+            <td>XYZ</td>
+            <td>2050</td>
+            <td>95%</td>
+          </tr>
+        </table>
+      </div>
+      
+
```
Comment (Rasagnya-Gurrala) [inline index.html:133]: add separate section for activities

**Review by Rasagnya-Gurrala — CHANGES_REQUESTED** [review]

**Review by AjitSharmaSBITSAccount — COMMENTED** [review]
Please add necessary sections like Your Projects  in the next commit.

File: index.html (hunk 1 above)
Comment (DeliaMDsouza) [reply index.html:133]: Added separate section for activities.

**Review by Rasagnya-Gurrala — COMMENTED** [review]
👍

**Review by Rasagnya-Gurrala — APPROVED** [review]

**Review by AjitSharmaSBITSAccount — APPROVED** [review]

**Pull request merged**